### PR TITLE
Reorganize for install via setup.py

### PR DESCRIPTION
**Pull request by craig-willis**
Not sure if you'd be interested in this.  I've been porting much of my temporal IR work to a Python-centric framework (based around [pyndri](https://github.com/cvangysel/pyndri/) and [parsl](https://github.com/Parsl/parsl), among other things.  I wanted to use your CV work and thought it might be handy to package for installation via setup.py or eventually pip. I made the bold assumption that this would be in the very stale "retrievable" namespace we once discussed...

In short:
* Moved modules into retrievable/cv subdirectory
* Added setup.py including console script
* Added a placeholder license, because might want one.

To test:
`pip install -e .` or `python setup.py install`

then
`run-cv ...`

And we should chat someday...

### CODE REVIEW

**Comment by gtsherman** [comment]
This seems great to me (also, thanks for pointing out pyndri, seems really useful). I certainly approve of putting it under the retrievable namespace.

I don't have experience with namespacing for Python, so I could be off-base, but I was looking at this [namespacing guide](https://packaging.python.org/guides/packaging-namespace-packages/) and it looks like the `pkg_resources` approach you've used in \_\_init\_\_.py is outdated. It looks like that should be changed to

```python
__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Other than that, I am totally happy to merge this.
